static/app/views/performance/transactionSummary/content.tsx

```diff
@@ -197,10 +191,62 @@ class SummaryContent extends React.Component<Props, State> {
         })
       );
 
-    const durationTableTitle =
-      spanOperationBreakdownFilter === SpanOperationBreakdownFilter.None
-        ? t('total duration')
-        : `${spanOperationBreakdownFilter} duration`;
+    const transactionsListTitles = organization.features.includes('trace-view-summary')
+      ? [t('event id'), t('user'), t('total duration'), t('trace id'), t('timestamp')]
+      : [t('event id'), t('user'), t('total duration'), t('timestamp')];
+
+    let transactionsListEventView = eventView.clone();
+
+    if (organization.features.includes('performance-ops-breakdown')) {
+      // update search conditions
+
+      const spanOperationBreakdownConditions = filterToSearchConditions(
+        spanOperationBreakdownFilter,
+        location
+      );
+
+      if (spanOperationBreakdownConditions) {
+        eventView = eventView.clone();
+        eventView.query = `${eventView.query} ${spanOperationBreakdownConditions}`.trim();
+        transactionsListEventView = eventView.clone();
+      }
+
+      // update header titles of transactions list
+
+      const operationDurationTableTitle =
+        spanOperationBreakdownFilter === SpanOperationBreakdownFilter.None
+          ? t('operation duration')
+          : `${spanOperationBreakdownFilter} duration`;
+
+      // add ops breakdown duration column as the 3rd column
+      transactionsListTitles.splice(2, 0, operationDurationTableTitle);
+
+      // span_ops_breakdown.relative is a preserved name and a marker for the associated
+      // field renderer to be used to generate the relative ops breakdown
+      let durationField = SPAN_OP_RELATIVE_BREAKDOWN_FIELD;
+
+      if (spanOperationBreakdownFilter !== SpanOperationBreakdownFilter.None) {
+        durationField = filterToField(spanOperationBreakdownFilter)!;
+      }
+
+      const fields = [...transactionsListEventView.fields];
+
+      // add ops breakdown duration column as the 3rd column
+      fields.splice(2, 0, {field: durationField});
```

Comment:
Is order here important? Just a push would suffice right?

Reply:
@Zylphrex yea order is important here.

Reply:
The `fields` correspond to the column order on the transactions list.